test-requirements.txt

```diff
@@ -1,2 +1,3 @@
 pytest~=7.2.2
 coverage~=7.2.2
+testfixtures~=8.2.0
```

Comment:
We are introducing a new dependency here, why it was needed?

Reply:
This is needed as we are mocking the logger in the tests.

Reply:
ok